ReleaseNotes.rst

```diff
@@ -7,6 +7,7 @@ Next Release
 
 - float / number type now support range restrictions
 - ranges on non number types (e.g. seq, string) now need to be non negative.
+- Fixed regex issue #22 and added regex unit tests.
```

Comment:
Please try to make this line more descriptive what problem was fixed and there is no need to link to the issue in the release notes.